scripts/ci/metrics.sh

```diff
@@ -159,10 +159,12 @@ WHERE
 GROUP BY
     Classname,
     Name
+HAVING
+    COUNTIF(Status="failed") > 0
```

Comment:
I think we need to take into account how many times the test failed. E.g. intializationError will have always 100% failure rate, the same with suspicious logs. So maybe we need check only things that has more than average number of runs?

Reply:
Good catch, and a good idea for a solution too! Although it's not obvious to me how to achieve this, I guess this will be an analytical function? :thinking: